### ISSUE TITLE
governor of mod relation

### ISSUE THREAD

**Issue by bguil** [surfacesyntacticud/guidelines#19]
When a relation `mod` has a `AUX`+`VERB` as governor, `AUX` is more often the head but it seems to be difficult to decide.
For instance, with post modifier:

|              | head=AUX | head=VERB |
|--------|:----------:|:-----------:|
| SUD_French-Spoken | [181](http://match.grew.fr/?corpus=SUD_French-Spoken@latest&custom=5ff5f1194a5d1) | [61](http://match.grew.fr/?corpus=SUD_French-Spoken@latest&custom=5ff5f0c116e86) |
| SUD_French-GSD | [3000+](http://match.grew.fr/?corpus=SUD_French-GSD@latest&custom=5ff5f1194a5d1) | [589](http://match.grew.fr/?corpus=SUD_French-GSD@latest&custom=5ff5f0c116e86) |

I don't see a clear criterion to decide. Should we make it more consistent?

**Comment by perrier54** [comment]
For French, the distinction is linguistically justified. When a modifier is attached to the auxiliary, it means that it is a sentence modifier. When the modifier is attached to the verb, it means it is a verb modifier.

> Examples: `Aujourd'hui, j'ai  travaillé (Today, I have worked)` - sentence modifier
`J'ai travaillé dur (I worked hard)`- verb modifier

**Comment by kimgerdes** [comment]
I agree that there can be a linguistically relevant distinction between higher and lower attachment, but I think that we need tests.
The test of dislocation of French seems to work but it doesn't generalize well to other languages, and somehow, semantically, I don't see that `aujourd'hui` is less connected to `travaillé` than `dur`. We could say that time and place modifiers connect to the highest verb of verbal chain. But this might be too semantic as criteria...

**Comment by perrier54** [comment]
**Decision**: A modifier of a set `AUX + VERB`is attached to the auxiliary when it is a sentence modifier, otherwise it is attached to the verb.

**Correction of treebanks**:

- [ ] GSD
- [ ] Rhapsodie
- [ ] Paris Stories
